Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AppCenterContentPage.xaml.cs

```diff
@@ -11,6 +11,8 @@ namespace Contoso.Forms.Puppet
     [Android.Runtime.Preserve(AllMembers = true)]
     public partial class AppCenterContentPage : ContentPage
     {
+        const string LogTag = "AppCenterXamarinPuppet";
```

Comment:
may be `AppCenterXamarinFormsPuppet`? Otherwise we will have the same logs for two puppet apps.

Reply:
Quite optional, let's keep it as is